src/components/Footer/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,13 @@
+import './filter.scss';
+import { FilterItem } from '../FilterItem';
+import { allFilters } from '../../../../constants';
+
+export const FilterContainer: React.FC = () => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      {allFilters.map(filter => {
+        return <FilterItem value={filter} key={filter} />;
+      })}
```

Comment:
Let's use a short form
```suggestion
      {allFilters.map(filter =>  <FilterItem value={filter} key={filter} />)}
```